components/twitter/my-liked-tweets.js

```diff
@@ -0,0 +1,25 @@
+const twitter = require('https://github.com/PipedreamHQ/pipedream/components/twitter/twitter.app.js')
+
+module.exports = { 
+  name: "my-liked-tweets", 
+  version: "0.0.1",
+  props: {
+    db: "$.service.db",
+    twitter,
+    timer: {
+      type: "$.interface.timer",
+      default: {
+        intervalSeconds: 60 * 15,
+      },
+    },
+  },
+  dedupe: "unique",
+  async run(event) {     
+    (await this.twitter.getLikedTweets()).reverse().forEach(tweet => {
```

Comment:
can you set `ts`? if so don't need reverse

Reply:
unfortunately, there's no timestamp when the "like" action occurred. the only timestamp is when the liked tweet was initially authored.